content/en/docs/reference/command-line-tools-reference/feature-gates/api-self-subject-review.md

```diff
@@ -4,6 +4,19 @@ content_type: feature_gate
 _build:
   list: never
   render: false
+
+stages:
+  - stage: alpha
+    fromVersion: "1.26"
+    toVersion: "1.26"
+    featureGateDefault: false
+  - stage: beta
+    fromVersion: "1.27"
+    fromVersion: "1.27"
+    featureGateDefault: true
+  - stage: stable
+    fromVersion: "1.28"
+    featureGateDefault: true
```

Comment:
Included to show that the demo doesn't include this graduated feature gate in the list of alpha and beta features.